### DISCUSSION TITLE
로딩씬을 이용한 씬 이동 개발 중 오류 발생

### DISCUSSION THREAD

**Discussion by DoLongJu** [naverz/zepeto-studio-kor#133] [5. Q&A]
아래 사이트를 보고 로딩씬을 만들어보던 중 아래 사진과 같은 오류가 발생됩니다.
오브젝트 충돌 인식 후 로딩씬을 불러오는 방식으로 만들고 있었습니다.
인풋모듈 오류 같은데 어떻게 해결해야 할까요?

https://wergia.tistory.com/5

![KakaoTalk_20211223_150329833](https://user-images.githubusercontent.com/96557169/147201215-dc4ad6f3-e9ff-4aae-9339-4700d2b5da93.png)
![1640242766](https://user-images.githubusercontent.com/96557169/147201443-d763e79d-0fe5-4a5b-81e1-c25f056905a0.png)

**Comment by DoLongJu** [comment]
로딩 씬에도 ZepetoPlayers를 포함해주면 해결됩니다!

**Comment by moondory77-naverz** [comment]
안녕하세요.
언급주신 내용과 관련하여, 편의를 위해 
씬 전환 시, ZepetoPlayers 컴포넌트의 DontDestroyOnLoad 옵션을 추가 제공드릴 수 있도록 하겠습니다.
감사합니다.
